CONTRIBUTING.md

```diff
@@ -117,6 +117,65 @@ usually a good idea to first open an issue describing the change to solicit
 feedback and guidance. This will increase the likelihood of the PR getting
 merged.
 
+
+## Steps to follow for getting started
+Note: If you are a beginner and wondering what set of steps you should
+follow for getting started with contributing after you have found an issue, these
+are the recommended steps.
+
+1. [Fork](https://guides.github.com/activities/forking/) the main tokio repository
+and create your own branch with a suitable name(so that it's easy to recognise the branch
+later when raising a pull request).
+
+2. Clone your project, switch branches, and make the required changes locally.
+
+3. As mentioned above, Tokio uses features and for most cargo commands a `--all-features`
+flag needs to be added. So firstly check whether the project is compiling.
+
+    ```
+    cargo check --all-features
+    ```
+
+4. Rust has 2 types of tests, namely [integration tests](https://doc.rust-lang.org/rust-by-example/testing/integration_testing.html) and [documentation tests](https://doc.rust-lang.org/rustdoc/documentation-tests.html).
```

Comment:
Actually, Rust has three types of tests. However Tokio mainly uses only two of them.